stable/aws-calico/templates/crd.yaml

```diff
@@ -0,0 +1,238 @@
+# Create all the CustomResourceDefinitions needed for
+# Calico policy-only mode.
+
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
```

Comment:
This will not work with Helm v3, you need to place the CRDs in the `crds` dir next to templates, run `kustomize create --autodetect`,  commit the kustomization file, add kustomize.yaml to .helmignore, add `kubectl apply -k URL` to readme for Helm v2 users. See https://github.com/aws/eks-charts/tree/master/stable/appmesh-controller#installing-the-chart

Reply:
changed